recipe/meta.yaml

```diff
@@ -10,13 +10,14 @@ source:
   sha256: 52aa0d383d42360f4faa0ae9537ba2ca348eeab4db5f2dfd6343192d0ff4b833
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage('gklib', max_pin='x.x') }}
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - make
```

Comment:
Could you quickly comment on why this is added for posterity?

Reply:
Sure, `cmake` just calls `make` for actually building the project. For some reason, `make` gets installed automatically on many platforms, but not on `ppc64le`.